Encapsulation Exercises/temperature-converter/src/Temperature.java

```diff
@@ -0,0 +1,26 @@
+public class Temperature {
```

Comment:
use a more descriptive class name like` TemperatureConverter`

Reply:
change the base class from Temperature to TemperatureConverter. Leave the Main class as is